### ISSUE TITLE
Ecto lesson on changesets could use the Example.Person module instead of the User module

### ISSUE THREAD

**Issue by Exadra37** [elixirschool/elixirschool#1756]
The lesson about [Ecto Changesets](https://elixirschool.com/en/lessons/ecto/changesets/#creating-your-first-changeset) starts with defining the **User** module:

> Let’s see a common User schema:
>
```elixir
defmodule User do
  use Ecto.Schema

  schema "users" do
    field(:name, :string)
  end
end
```

But in the [previous lesson](https://elixirschool.com/en/lessons/ecto/basics/#schemas) we have created the **Person** module:

> Let’s create our new schema at lib/friends/person.ex:

```elixir
defmodule Example.Person do
  use Ecto.Schema

  schema "people" do
    field :name, :string
    field :age, :integer, default: 0
  end
end
```
In my opinion the **Person** module could be used instead of the **User** module in the Ecto changesets lesson.

**Comment by doomspork** [comment]
@Exadra37 this is a good suggestion, I _thought_ we'd done that when initially bringing the lessons together but evidently we did not 😁 

Would you be open to making this change in a PR for us?

**Comment by Exadra37** [comment]
Yes, I will try to find some time for it... Cannot promise exactly when.

Feel free to ping me if I start to take to long... sometimes I forget things, because I have so many to do, and only 24 hours in a day ;)

**Comment by doomspork** [comment]
No worries @Exadra37 I understand we can all get busy 😁

**Comment by SophieDeBenedetto** [comment]
Closing since this seems to be resolved by #2011
